Directory.Build.props

```diff
@@ -54,10 +54,22 @@
     <ForceGenerationOfBindingRedirects>true</ForceGenerationOfBindingRedirects>
     <GenerateDocumentationFile>true</GenerateDocumentationFile>
     <ProduceReferenceAssembly>true</ProduceReferenceAssembly>
-    <!-- TODO: Might be related to: https://github.com/dotnet/project-system/issues/5619 -->
-    <EnableSourceLink Condition="'$(CIBuild)' == 'true'">true</EnableSourceLink>
     <NoWarn>$(NoWarn);NU5125</NoWarn>
 
+    <!-- SourceLink properties -->
+    <!-- https://github.com/dotnet/sourcelink#using-source-link-in-net-projects -->
+    <EnableSourceLink>true</EnableSourceLink>
+    <!-- Publish the repository URL in the built .nupkg (in the NuSpec <Repository> element) -->
+    <PublishRepositoryUrl>true</PublishRepositoryUrl>
+    <!-- Embed source files that are not tracked by the source control manager in the PDB -->
+    <EmbedUntrackedSources>true</EmbedUntrackedSources>
+    <!-- Build symbol package (.snupkg) to distribute the PDB containing Source Link -->
+    <IncludeSymbols>true</IncludeSymbols>
+    <SymbolPackageFormat>snupkg</SymbolPackageFormat>
+    <!-- Marks the assembly be built in a determinstic way. -->
+    <!-- See: https://github.com/dotnet/sdk/issues/16325#issuecomment-981818164 -->
+    <ContinuousIntegrationBuild>true</ContinuousIntegrationBuild>
```

Comment:
I didn't make any of these conditional on `CIBuild` as there didn't seem to be a reason to not enable this functionality for local builds too.

Reply:
When you build locally, it is convenient that the .pdbs point to your local source locations on disk, so finding local files is easy during debugging. My understanding is that setting these locally will normalize the paths to source files to not contain your local paths, so VS debugger might find it difficult to find local sources.

Reply:
Alright. I'll change it and put that exact information there. You know how difficult it is to find what a single MSBuild property does? MSBuild desperately needs a documentation (IntelliSense-style) system built-in for these things. There hundreds and hundreds of properties in a binlog. Trying to figure out what each does is madness. I'm commenting on these things as much as possible at this point because I won't remember 6 months from now.

Reply:
I usually search the preprocessed project text for the property name (or use Find in Files feature). It is hard indeed, so good job commenting it.